pkg/plugins/optional/grafana/v1alpha/scaffolds/internal/templates/runtime.go

```diff
@@ -105,6 +105,62 @@ const controllerRuntimeTemplate = `{
       "title": "Reconciliation Metrics",
       "type": "row"
     },
+    {
+      "datasource": "${DS_PROMETHEUS}",
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [],
```

Comment:
Should we consider having a way to preview the graphs in the CI / GitHub actions before merging this one? So that we can check more efficiently if all is still fine. WDYT?